VideoApp/VideoApp/Video/Tracks/Camera/CameraConfigFactory.swift

```diff
@@ -36,13 +36,17 @@ class CameraConfigFactory {
             
             cropRatio = CGFloat(targetSize.width) / CGFloat(targetSize.height)
             frameRate = 20
-        case .vp8Simulcast:
+        case .vp8SimulcastVGA:
+            targetSize = CMVideoDimensions(width: 640, height: 480)
+            cropRatio = CGFloat(targetSize.width) / CGFloat(targetSize.height)
+            frameRate = 24 // With VGA simulcast enabled there are 3 temporal layers, allowing a frame rate of {f, f/2, f/4}
+        case .vp8SimulcastHD:
```

Comment:
Can we use the same cropping as .h264 and .vp8? Just looking to make Vp8 simulcast less power hungry if this will be the default option.

Reply:
To be clear `vp8SimulcastVGA` will be the default option so changing this line will not have an impact on default. Easy to change though...still desired?

Reply:
Yes, on line 40.

Reply:
Oh hmmm but if we use `targetSize = CMVideoDimensions(width: 544, height: 480)` for simulcast VGA we will be different than documented best practices. Still go for it?

Reply:
I pushed changes but please confirm we want to use width that is a little different than documented `640`.